public/src/client/topic.js

```diff
@@ -143,7 +143,7 @@ define('forum/topic', [
 			if (components.get('post/anchor', postIndex).length) {
 				return navigator.scrollToPostIndex(postIndex, true);
 			}
-		} else if (bookmark && (!config.usePagination || (config.usePagination && ajaxify.data.pagination.currentPage === 1)) && ajaxify.data.postcount > 5) {
+		} else if (bookmark && (!config.usePagination || (config.usePagination && ajaxify.data.pagination.currentPage === 1)) && ajaxify.data.postcount > ajaxify.data.config.bookmarkThreshold) {
```

Comment:
`ajaxify.data.config`, or just `config`?

I've always used the latter, though technically, window globals are a no-no... @barisusakli should I be using the former?